feat(COC): add coc script orders data source (#6818)
huaweicloud/terraform-provider-huaweicloud: docs/data-sources/coc_script_orders.md

---
subcategory: "Cloud Operations Center (COC)"
layout: "huaweicloud"
page_title: "HuaweiCloud: huaweicloud_coc_script_orders"
description: |-

>   Use this data source to query the job service ticket list.

Reviewer: ```suggestion
  Use this data source to query the script orders list.
```

---

# huaweicloud_coc_script_orders

Use this data source to query the job service ticket list.

## Example Usage

```hcl
data "huaweicloud_coc_script_orders" "test" {}
```

## Argument Reference

The following arguments are supported:

* `start_time` - (Optional, Int) Specifies the start time.

* `end_time` - (Optional, Int) Specifies the end time.

* `creator` - (Optional, String) Specifies the creator.

* `status` - (Optional, String) Specifies the service ticket status.
  Values can be as follows:
  + **READY**: Prepare.
  + **PROCESSING**: The operation is in progress.
  + **ABNORMAL**: Abnormal.
  + **PAUSED**: Paused.
  + **CANCELED**: Canceled.
  + **FINISHED**: Success.

## Attribute Reference

In addition to all arguments above, the following attributes are exported:

* `id` - The data source ID.

> * `data` - Indicates the list of work order.

Reviewer: ```suggestion
* `data` - Indicates the list of script orders.
```

  The [data](#data_struct) structure is documented below.

<a name="data_struct"></a>
The `data` block supports:

* `order_id` - Indicates the primary key ID.

> * `order_name` - Indicates the service ticket name.

Reviewer: ```suggestion
* `order_name` - Indicates the script order name.
```

* `execute_uuid` - Indicates this UUID is used when the list is redirected to the details page.

* `gmt_created` - Indicates the creation time.

* `gmt_finished` - Indicates the completion time.

* `execute_costs` - Indicates the execution duration in seconds.

* `creator` - Indicates the creator.

> * `status` - Indicates the service ticket status.

Reviewer: ```suggestion
* `status` - Indicates the script order status.
```

---
subcategory: "Cloud Operations Center (COC)"
layout: "huaweicloud"
page_title: "HuaweiCloud: huaweicloud_coc_script_orders"
description: |-
  Use this data source to query the job service ticket list.
---

# huaweicloud_coc_script_orders

Use this data source to query the job service ticket list.

## Example Usage

```hcl
data "huaweicloud_coc_script_orders" "test" {}
```

## Argument Reference

The following arguments are supported:

* `start_time` - (Optional, Int) Specifies the start time.

* `end_time` - (Optional, Int) Specifies the end time.

* `creator` - (Optional, String) Specifies the creator.

* `status` - (Optional, String) Specifies the service ticket status.

>   Values can be as follows:

Reviewer: Missing ready, check the API docs and confirm it.

  + **PROCESSING**: The operation is in progress.
  + **ABNORMAL**: Abnormal.
  + **PAUSED**: Paused.
  + **CANCELED**: Canceled.
  + **FINISHED**: Success.

## Attribute Reference

In addition to all arguments above, the following attributes are exported:

* `id` - The data source ID.

* `data` - Indicates the list of work order.

  The [data](#data_struct) structure is documented below.

<a name="data_struct"></a>
The `data` block supports:

* `total` - Indicates the total number of rules.

* `data` - Indicates the data list.

  The [data](#data_data_struct) structure is documented below.

<a name="data_data_struct"></a>
The `data` block supports:

> 

Reviewer: does not match the schema you designed
